g3doc/quick_reference.md

```diff
@@ -636,6 +636,19 @@ All other ops in this section are only available if `HWY_TARGET != HWY_SCALAR`:
     Exception: if `HWY_TARGET == HWY_SCALAR`, returns `a[0]*b[0]`. Note that the
     initial value of `sum1` must be zero, see `ReorderWidenMulAccumulate`.
 
+*   `VN`: `{u,i}{8,16}`, `D`: `Repartition<MakeWide<MakeWide<TFromV<VN>>>>` \
```

Comment:
Nit: `RepartitionToWide<RepartitionToWide<DFromV<VN>>>`. Let me know if you'd like me to change this.

Reply:
Updated documentation